crates/muvm/src/guest/bin/muvm-guest.rs

```diff
@@ -77,20 +77,25 @@ fn main() -> Result<()> {
     rustix::stdio::dup2_stdout(console.as_fd())?;
     rustix::stdio::dup2_stderr(console.as_fd())?;
 
-    Command::new("/usr/lib/systemd/systemd-udevd").spawn()?;
+    Command::new(std::option_env!("MUVM_UDEVD_PATH").unwrap_or("/usr/lib/systemd/systemd-udevd"))
```

Comment:
Any particular reason why these should be compile time?

Reply:
They were compile time before and I personally haven't had a reason to add runtime configurability there so far. For Nix we just want an easier compile-time override for paths that get saved into the binary, nothing more.

Reply:
Added a flag for overriding at runtime (might actually want that myself after all)